### ISSUE TITLE
Discrepancy between Eq. (10) in the paper and loss calculation in the code

### ISSUE THREAD

**Issue by hourlass** [ZhangXiaotong015/GATSegDiff#3]
Hi authors,
Thank you for your impressive work and for sharing the code!I am currently reproducing your work based on the paper and this repository. I noticed a discrepancy regarding the total loss function definition between the paper and the implementation.In the paper (specifically Eq. 7 and Eq. 10), the Graph Loss ($L_{graph}$) is explicitly included in the total loss function:
$$L_{total} = L_{den} + L_{graph}$$
However, in gaussian_diffusion.py, I noticed that while loss_cal (which I assume corresponds to $L_{graph}$) is calculated, it seems to be excluded from the final total loss terms["loss"].
Was the auxiliary graph loss found to be unstable or unnecessary during your final experiments?

`# ... inside training_losses_segmentation ...

# loss_cal is calculated here
terms["loss_cal"] = F.binary_cross_entropy_with_logits(
    cal.view(*model_kwargs['speed'].shape), 
    model_kwargs['speed']
)

if "vb" in terms:
    # loss_cal is NOT added here
    terms["loss"] = terms["mse_diff"] + terms["vb"] 
else:
    # loss_cal is NOT added here either
    terms["loss"] = terms["mse_diff"]`

**Comment by ZhangXiaotong015** [comment]
Thanks for your question. You're right that loss_cal corresponds to L_graph. The graph loss was added to the mse loss before back propogation. You can find the loss combination in '/guided_diffusion/train_util.py' as the following figure shows.

<img width="823" height="193" alt="Image" src="https://github.com/user-attachments/assets/9330d80e-3e9e-45ac-b962-947962db97ed" />
